trust/extract-jks.c

```diff
@@ -247,10 +249,38 @@ prepare_jks_buffer (p11_enumerate *ex,
 	 * when this was this certificate was added to the keystore, however
 	 * we don't have that information. Java uses time in milliseconds
 	 */
-	if (_p11_extract_jks_timestamp)
-		now = _p11_extract_jks_timestamp;
-	else
-		now = time (NULL);
+	{
+		char *source_date_epoch;
+		source_date_epoch = getenv("SOURCE_DATE_EPOCH");
```

Comment:
Please use `secure_getenv` (it's not a problem here, but for consistency), and insert a space before the opening parenthesis.